modules/module_shell_test.go

```diff
@@ -49,11 +50,13 @@ func TestShell(t *testing.T) {
 	sQuiet := &ShellModule{cfg: &config.Config{Verbose: false}}
 	sVerbose := &ShellModule{cfg: &config.Config{Verbose: true}}
 
+	env := environment.New()
+
 	// Arguments
 	args := make(map[string]interface{})
 
 	// Rn with no arguments to see an error
```

Comment:
Not your fault, but if you're changing this section of code anyway "Rn -> Run" :)

Reply:
No worries, done.